content-review/impl/turnitin-oc/src/main/java/org/sakaiproject/contentreview/turnitin/oc/ContentReviewServiceTurnitinOC.java

```diff
@@ -512,6 +514,9 @@ public String getReviewReportRedirectUrl(String contentId, String assignmentRef,
 				modes.put(MATCH_OVERVIEW, Boolean.TRUE);
 				modes.put(ALL_SOURCES, Boolean.TRUE);
 				similarity.put(MODES, modes);
+				Map<String, Object> viewSettings = new HashMap<>();
```

Comment:
Map<String, Object> viewSettings seems to only ever store Boolean types, does Map<String, Boolean> would seem ideal as it will avoid casting.

Reply:
For now, there is only a boolean value, but "view_settings" is just a generic JSON object and could have any JSON object in the future (we only expose certain options in the API as needed)